models/LifeJourney.go

```diff
@@ -0,0 +1,10 @@
+package models
+
+type LifeJourney struct {
+	ID              int      `json:"id"`
+	Title           string   `json:"title"`
+	Description     string   `json:"description"`
+	Title_fr        string   `json:"title_fr"`
```

Comment:
We should try to keep a standard, maybe use TitleFr or TitleFrench to make it easy to read?